app/services/resource_filter.rb

```diff
@@ -70,6 +70,12 @@ def filter(collection)
         proposal_in_meetings_ids = MeetingProposal.pluck(:proposal_id).uniq
         collection = collection.where(id: proposal_in_meetings_ids)
       end
+
+      if @params["source"] && @params["source"].include?("organization")
+        proposal_from_organizations_ids = Organization.verified.map{ |o| o.user.proposals.ids }.flatten.uniq
```

Comment:
I don't like this that much but I think it works. The query I'm looking for is "Proposals only from verified organizations"